src/LoggerConfig.ts

```diff
@@ -30,6 +30,10 @@ export default class LoggerConfig {
         return this.level;
     }
 
+    public captureStack():boolean {
+        return this.capture_stack;
```

Comment:
The good idea to pass not only flag, but minimal level for displaying stacktraces. For example, you could print stacktraces for WARNING level and higher.